### DISCUSSION TITLE
AttributeError: module 'pyvista' has no attribute 'rcParams'

### DISCUSSION THREAD

**Discussion by ZhaoYu810** [pyvista/pyvista#6690] [Q&A] [answered]
### First check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the PyVista documentation, with the integrated search.
- [X] I already searched in Google "How to X in PyVista" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I will NOT make duplicate discussions or issues in the future.

### Commit to Help

- [X] I commit to help with one of those options.

### Sample Code What is the problem, question, or error?
Hello everyone. 
Recently, I cloned a project named "DeepField" from GitHub. When running it, the following error occurred: "AttributeError: module 'pyvista' has no attribute 'rcParams'".

According to my previous experience, I have updated and reinstalled pyvista many times, but I still cannot solve this problem. On Google, it seems that everyone is using "pv.rcParams" normally.

The following are the error-prone statements located according to PyCharm.

```python
  pv.rcParams["camera"]["viewup"] = [0, 0, -1]
  pv.rcParams["camera"]['position'] = [1, 1, -0.3]
```

### Description

The error "**AttributeError: module 'pyvista' has no attribute 'rcParams'**" occurs. 
I really don't understand why this problem occurs. I have reinstalled the pyvista library many times.

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Sun Oct 06 21:03:18 2024 

                OS : Windows (10 10.0.19045 SP0 Multiprocessor Free)
            CPU(s) : 12
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.9 GiB
       Environment : Python
        GPU Vendor : NVIDIA Corporation
      GPU Renderer : NVIDIA GeForce RTX 4060 Ti/PCIe/SSE2
       GPU Version : 4.5.0 NVIDIA 561.09
  MathText Support : False

  Python 3.12.7 | packaged by conda-forge | (main, Oct  4 2024, 15:47:54) [MSC
  v.1941 64 bit (AMD64)]

           pyvista : 0.44.1
               vtk : 9.3.0
             numpy : 2.0.2
        matplotlib : 3.9.2
            scooby : 0.10.0
             pooch : 1.8.2
            pillow : 10.4.0
           imageio : 2.35.1
             PyQt5 : 5.15.11
           IPython : 8.28.0
        ipywidgets : 8.1.5
             scipy : 1.14.1
              tqdm : 4.66.5
--------------------------------------------------------------------------------
```

### Screenshots

![image](https://github.com/user-attachments/assets/cfb1b19c-82c2-496b-bf82-49a6b88a230a)

**Comment by tkoyama010** [comment, marked as answer]
Upon reviewing the [DeepField](https://github.com/Skoltech-CHR/DeepField) repository, it appears that PyVista version 0.23.1 is supported. The API has undergone significant changes between versions 0.23.1 and 0.44.1. Please try using PyVista version 0.23.1.
